app/posts/register-trainee-teachers/2021-08-03-highest-academic-qualification.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Removing the highest academic qualification question
+description: Why we don’t think we need this question and an alternate design if we do.
+date: 2021-08-03
+---
+
+For trainees on undergraduate courses the provider has been asked for the trainee’s highest academic qualification.
+
+The question was causing users confusion. It seems to be because it’s inconsistent with how users think about academic qualifications. A teacher training provider thinks about a trainee’s qualifications as the measure of whether a trainee is qualified for their course and often what their ‘tariff points’ value is. In research sessions users tried to give the trainee’s actual qualifications. When users found they could only provide a single qualification most users re-read the question and were able to give the answer the service wanted.
+
+![Highest academic qualification — first iteration.](highest-academic-qualification-v1.png "Screenshot of ‘What is the trainee’s highest academic qualification?’ question and answers")
```

Comment:
I think you may have alt text and caption the wrong way round.

Reply:
Yep – fixed, thanks!